### ISSUE TITLE
Add i18n to '[locale]:' strings to allow custom spacing before colon

### ISSUE THREAD

**Issue by pedro-mendonca** [jrfnl/debug-bar-localization#26]
Hi, thank you for your awesome plugin!

I've found something that I think I can help with.
As you can see below, the colon after the Locale has no space after en_US and have a space after all the other locales.

![imagem](https://user-images.githubusercontent.com/7371591/210552471-fc66ea38-b408-4431-acea-8b118d88eaa8.png)

Most locales use no space between the last word and the colon, but some add a space, like French.
So, as a good practice, this should be translatable, can be as `%s:` so the translators can add a space if needed for their locale, can also add on other strings of the plugin, like `Installed Languages:`.

Would you like a PR with this fix? :)

**Comment by jrfnl** [comment]
@pedro-mendonca I agree that's inconsistent. A PR to fix it is always welcome.

Apologies in advance though. It may be a little while before I get to the PR. I'm also afraid that the CI will probably need an update before anything becomes mergable again, but I'll look at that when I get to it.

**Comment by pedro-mendonca** [comment]
Thanks, I'll push a quick fix, I also see that the code here is not up to date with w.org, but this is a very short fix, and probably won't be hard to deal with.

**Comment by jrfnl** [comment]
I agree, the change itself won't be hard to deal with, but the "fixing CI and updating the code for a new release" will be, so I won't be able to release the change for at least a couple of months.
